packages/core/contracts/Escrow.sol

```diff
@@ -229,11 +241,17 @@ contract Escrow is IEscrow, ReentrancyGuard {
         ) = finalizePayouts(_amounts);
 
         for (uint256 i = 0; i < _recipients.length; ++i) {
-            _safeTransfer(_recipients[i], finalAmounts[i]);
+            if (finalAmounts[i] > 0) {
```

Comment:
we might need to add require with this, the only for this to be 0 is that amount passed is 0 and this shouldn't happen

Reply:
ok, lets keep it as it is and I will ask Alex if we should add a limit of fees